internal/db/start/templates/schema.sql

```diff
@@ -1,10 +1,14 @@
 \set pgpass `echo "$PGPASSWORD"`
+\set jwt_secret `echo "$JWT_SECRET"`
+
+ALTER DATABASE postgres SET "app.settings.jwt_secret" TO :'jwt_secret';
```

Comment:
In theory this should offer the same result as https://github.com/supabase/supabase/pull/16681 but it requires SUPERUSER. Setting any unknown custom option at the db level requires superuser. On constrast, on https://github.com/supabase/supabase/pull/16681 postgREST just sets the custom option at the transaction level (doesn't require superuser).

Reply:
If the above is not a concern then yes it would be an alternative solution with the advantage of being able to access the JWT secret in any transaction like the one mentioned on https://github.com/supabase/supabase/pull/16681#pullrequestreview-1598681970

Reply:
Yup, for feature parity with platform, I will set it at the database level for now. We may change it in the future if this causes problem.